pymc3/distributions/continuous.py

```diff
@@ -23,12 +23,11 @@
 
 from aesara.assert_op import Assert
 from aesara.tensor.random.basic import (
-    beta,
+    BetaRV,
+    ExponentialRV,
+    GammaRV,
+    RandomVariable,
     cauchy,
-    exponential,
-    gamma,
-    halfcauchy,
-    halfnormal,
```

Comment:
I have a hard time believing that we need to recreate _all_ of these distributions.  Something's wrong here.

Reply:
The tests pass now (which was my goal). We can now rework from here and see what is needed or not. The original code before this PR was certainly not working yet.

Reply:
Yes, we're building toward complete conversion of the code.  It isn't going to happen in a single PR/commit.

It's very important that we not introduce new design issues just because we want to make things "work"/tests pass.  Remember, a big part of these `v4` changes involves fixing old design issues that most likely arose under similar circumstances.